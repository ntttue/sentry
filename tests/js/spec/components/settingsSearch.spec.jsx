import React from 'react';
import {shallow} from 'enzyme';
import SettingsSearch from 'app/components/settingsSearch';

describe('SettingsSearch', function() {
  beforeEach(function() {
    this.sandbox = sinon.sandbox.create();
  });

  afterEach(function() {
    this.sandbox.restore();
  });

  it('renders', function() {
    let wrapper = shallow(<SettingsSearch />);
    expect(wrapper).toMatchSnapshot();
  });
});
