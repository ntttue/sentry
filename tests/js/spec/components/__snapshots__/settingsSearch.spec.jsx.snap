// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SettingsSearch renders 1`] = `
<div
  className="settings-search"
/>
`;
